lib/common.sh

```diff
@@ -152,24 +152,40 @@ resolve_ipv4()
     local hname="$1"
 
     # Resolve hostname to IPv4 address.
-    host_op=$(host -4 -t A "$hname" | sort) 
+    host_op=$(host -4 -t A "$hname" | sort)
```

Comment:
for adding SCALABILITY support as discussed in the doc, we might have to do something as simple as

host_op=$(host -4 -t A "$hname" | sort | shuf --random-source=/etc/machine-id)